README.md

```diff
@@ -1,15 +1,17 @@
+[![Build Status](https://travis-ci.org/RUGSoftEng/2017-Hestia-Client.svg?branch=development)](https://travis-ci.org/RUGSoftEng/2017-Hestia-Client)
```

Comment:
Was going to ask for a badge in the readme, but now I see you already did so :+1: